src/pages/workspace/accounting/qbo/import/QuickbooksLocationsPage.tsx

```diff
@@ -69,9 +71,11 @@ function QuickbooksLocationsPage({policy}: WithPolicyProps) {
                             />
                         </OfflineWithFeedback>
                     )}
-                    <View style={[styles.flex1, styles.flexRow, styles.alignItemsCenter, styles.gap2, styles.mt1]}>
-                        <Text style={styles.mutedTextLabel}>{translate('workspace.qbo.locationsAdditionalDescription')}</Text>
-                    </View>
+                    {shouldBeDisabled && (
```

Comment:
Although related to the `Taxes` implementation and not directly connected to our implementation here, I think it is better to hide the hint text when `Journal Entry` is selected and the `Tax` is enabled due to an edge case. We have done this improvement here for `Locations` but somehow got missed in the other PR for Taxes.
![Screenshot 2024-05-27 at 8 19 13 AM](https://github.com/Expensify/App/assets/3004659/6a273f26-459e-4cf4-ac44-d28502bc01cb)

Reply:
https://github.com/Expensify/App/pull/41638/commits/06880889020ad67d95dbfcf6e05323c6efae96df

Reply:
@hayata-suenaga Oh! I was not suggesting to make changes in `QuickbooksLocationsPage`. Instead, I meant changing in `QuickbooksTaxesPage` so that it behaves like `QuickbooksLocationsPage` which seems correct. Sorry if my explanation was misleading.

Reply:
https://github.com/Expensify/App/pull/41638/commits/805b67ed39e7a8cd1c2064503942df5d366f9bad

Reply:
no your explanation was clear 🙇 I implemented your suggestion [here](https://github.com/Expensify/App/pull/41638/commits/a313d7a6d5ba5948fdedc261e0b5280f2fe01609)